packages/terra-paginator/src/ControlledProgressivePaginator.jsx

```diff
@@ -47,6 +47,12 @@ const propTypes = {
    * Allows user to set custom page label. _(usage note: User must pass translated text)_.
    */
   pageLabel: PropTypes.string,
+  /**
+   * ![IMPORTANT](https://badgen.net/badge/UX/Accessibility/blue)
+   * When specified allows user to set custom page count. User should provide custom page count as part `pageLabel` for best accessibility practices.
+   * _(usage note: when `pageLabel` is not provided page count will not be hidden and default page count is displayed for best accessibility practices)_.
+   */
+  hidePageCount: PropTypes.bool,
```

Comment:
Should we make this new prop private since it's only for Fusion usage?

Reply:
I think we decided to make this available for other consumers as well. hence keeping it available for all ( not just for fusion).